src/translator/CMakeLists.txt

```diff
@@ -31,8 +25,13 @@ if(COMPILE_WASM)
   # Enable code that is required for generating JS bindings
   target_compile_definitions(bergamot-translator PRIVATE WASM_BINDINGS)
   target_compile_options(bergamot-translator PRIVATE ${WASM_COMPILE_FLAGS})
+else(COMPILE_WASM)
+  target_compile_definitions(bergamot-translator PUBLIC WITH_PTHREADS)
```

Comment:
The sane platform should be the default (i.e. without defines). Besides, the absence of this flag does not completely remove pthreads.

Reply:
I have changed this from `#ifdef WITH_PTHREADS` -> `#ifndef WASM_HIDE_THREADS`.